infrastructure/main.tf

```diff
@@ -91,3 +105,42 @@ resource "azurerm_key_vault_secret" "error_notifications_username" {
 }
 
 # endregion
+
+# region DB secrets
+resource "azurerm_key_vault_secret" "POSTGRES-USER" {
+  name         = "${var.component}-POSTGRES-USER"
+  key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
+  value        = "${module.blob-router-db.user_name}"
+}
+
+resource "azurerm_key_vault_secret" "POSTGRES-PASS" {
+  name         = "${var.component}-POSTGRES-PASS"
+  key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
+  value        = "${module.blob-router-db.postgresql_password}"
+}
+
+resource "azurerm_key_vault_secret" "POSTGRES_HOST" {
+  name         = "${var.component}-POSTGRES-HOST"
+  key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
+  value        = "${module.blob-router-db.host_name}"
+}
+
+resource "azurerm_key_vault_secret" "POSTGRES_PORT" {
+  name         = "${var.component}-POSTGRES-PORT"
+  key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
+  value        = "${module.blob-router-db.postgresql_listen_port}"
+}
+
+resource "azurerm_key_vault_secret" "POSTGRES_DATABASE" {
+  name         = "${var.component}-POSTGRES-DATABASE"
+  key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
+  value        = "${module.blob-router-db.postgresql_database}"
+}
+
```

Comment:
Would be nice to not copy paste legacy `resources` because somebody in the past put down these values. Would be nice to expect next PR amending these points:

- resource names lowercase + underscores
- key vault names lowercase + hyphens

P.S. no need to reference `POSTGRESS`. It doesn't really matter which DB we use. It matters if component provides customisation to public. Since it doesn't - no matter.

P.S. name prefix I believe should just be `db`. If so, the reference to `postgres` could stay. So we could do as follows:

```
name = "db-postgres-user"
...
```